seed/data_importer/tasks.py

```diff
@@ -504,6 +504,8 @@ def _cleanse_data(file_pk, record_type='property'):
 def map_data(file_pk, *args, **kwargs):
     """Small wrapper to ensure we isolate our mapping process from requests."""
     _map_data.delay(file_pk)
+    prog_key = get_prog_key('map_data', file_pk)
+    return {'status': 'in process...', 'progress_key': prog_key}
```

Comment:
Did you want me to take out the status key?  Or change it to progress?  Or nothing?  The other conflicts are now taken care of, so if your answer is _nothing_ then this is ready to drop.